web/source/dom/domManager.ts

```diff
@@ -907,6 +922,30 @@ namespace com.keyman.dom {
       }
     }.bind(this);
 
+    _InputModeObserverCore = function(this: DOMManager, mutations: MutationRecord[]) {
```

Comment:
Can you add a comment explaining the purpose of this function? Also I think we should be naming it `_inputModeObserverCore`?

Reply:
The other observer core functions have similar capitalization:

- `_AutoAttachObserverCore`
- `_EnablementMutationObserverCore`

Reply:
> Should I change them, too?

It's probably good sanitisation to iteratively cleanup to javaScript camelCasing preference, so yeah, if it's not too big a yak shave, sure!